### DISCUSSION TITLE
连续几天无法正常使用，网络根本连不上，要求退款

### DISCUSSION THREAD

**Discussion by fbsz2** [getlantern/lantern-forum-cn#1088] [General]
460575560@qq.com
![微信图片_20241218093140](https://github.com/user-attachments/assets/e75ae8f3-70cf-45f3-ba8b-3f0426ec6d25)
![微信图片_20241218093140](https://github.com/user-attachments/assets/d40d50aa-9945-406a-abb4-33ee1a6468a1)

**Comment by MaxMKMax** [comment]
@fbsz2 

给您弄退款之前需要您帮一个忙. 不然无法帮您提交退款申请.

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

我们将非常感谢您。

**Reply by fbsz2** [reply]
80460286

**Comment by MaxMKMax** [comment]
@fbsz2 您好. 谢谢您提供的信息. 我已经帮您提交退款申请.了. 谢谢您支持蓝灯. 希望您以后会再回来使用Lantern. 欢迎您回来.
